src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadFileProcessor.java

```diff
@@ -244,6 +250,7 @@ private void processFilePath(BulkLoad bulkLoad, String localFilePath) {
             bulkLoadFile.setFileSize(inputFile.length());
             bulkLoadFile.setStatus(BulkLoadStatus.PENDING);
             bulkLoadFile.setLocalFilePath(localFilePath);
+            bulkLoadFile.setFileType(fileType);
```

Comment:
@markquintontulloch for instance this line would be a `bulkLoadFile.setFileExtension(bulkLoad.getFileExtension());

Reply:
@oblodgett - Makes sense, I'll revisit it tomorrow, thanks

Reply:
And then `bulkLoad.fileExtension` would be set in the UI when creating the BulkLoad, plus we need to considering the files that have been uploaded already. Those may need to be renamed in S3 to that the URLs continue to work.